pkgs/development/interpreters/dart/default.nix

```diff
@@ -11,24 +11,24 @@
     aarch64 = "arm64";
     # Make sure that if the user overrides version parameter they're
     # also need to override sources, to avoid mistakes
-    version = "2.10.5";
+    version = "2.12.2";
   in
   {
     "${version}-x86_64-darwin" = fetchurl {
       url = "${base}/stable/release/${version}/sdk/dartsdk-macos-${x86_64}-release.zip";
-      sha256 = "1vb2m25w6v901id9syan9q69fa60sxxd7qpyzq21fn5dpah0g99i";
+      sha256 = "0000000000000000000000000000000000000000000000000000";
```

Comment:
Something went wrong here with hashes.

Reply:
Ah yes. I've noticed it before pushing but forgot to mention after opening the pr.
I just changed the hashes to `000...` for all, then ran `nix-build -A dart` and put the hash from the error message back in as I usually do when updating packages. But since I only have an x86 linux, it only built for that. I guess I need to manually download the files for each arch, check the shasum and put that in the variables?

Reply:
Yeah, you need to download each of the files using `nix-prefetch-url <url>`.

It would be nice to have an auto-update script to do this.

Reply:
Should be fixed now.